02-how-to-use.Rmd

```diff
@@ -2,179 +2,95 @@
-feels slow. If you have experience as an educator or consultant, you already
-have efficient solutions you use in your day-to-day work. Introducing code to
-your workflow slows you down at first because you won't be as fast as you are
-with your favorite spreadsheet software. However, you're probably reading this book because you realize that learning how to analyze data using R is like investing in your own personal infrastructure---it takes time while you're building the initial skills, but the investment pays off when you start solving complex problems faster and at scale. One person we spoke with shared this story about their learning journey:
+The beginning of the learning journey is particularly challenging because it feels slow. In your current work environment, you already have efficient solutions you use to accomplish your day-to-day work. Introducing code to your workflow will slow you down at first because you won't be as fast as you are with your favorite spreadsheet software. However, you're probably reading this book because you realize that learning how to analyze data using R is like investing in your own personal infrastructure---it takes time while you're building the initial skills, but the investment pays off when you start solving complex problems faster and at scale. One person we spoke with shared this story about their learning journey:
 
 > The first six months were hard. I knew how quickly I could do a pivot table
 > in Excel. It took longer in R because I had to go through the syntax and take
 > the book out. I forced myself to do it, though. In the long term, I'd be a
 > better data scientist. I'm so glad I thought that way, but it was hard the
 > first few months.
 
-Our message is this: learning R for your education job is doable, challenging,
-and rewarding all at once. We wrote this book for you because we do this work
-every day. We're not writing as education data science masters. We're writing as
-people who learned R and data science *after* we chose education. And like you,
-improving the lives of students is our daily practice. Learning to use R and data
-science helped us do that. Join us in enjoying all that comes with R and data science---both the challenge of
-learning and the joy of solving problems in creative and efficient ways.
+Our message is this: learning R for your education job is doable, challenging, and rewarding all at once. We wrote this book for you because we do this work every day. We're not writing as people who have mastered all there is to know about education data science. We're writing as people who learned R and data science *after* we chose education. And like you, improving the lives of students is our daily practice. Learning to use R and data science helped us do what matters most to us. Join us in enjoying all that comes with R and data science---both the challenges of learning and the joys of solving problems in creative and efficient ways.
```

Comment:
maybe change to "education-related job" or "role in education"? Just thinking some in higher education will not chiefly see themselves as having an "education job".